Sources/KituraWebSocket/WSConnectionUpgradeFactory.swift

```diff
@@ -34,14 +34,26 @@ public class WSConnectionUpgradeFactory: ProtocolHandlerFactory {
     /// Return a WebSocketConnection channel handler for the given request
     public func handler(for request: ServerRequest) -> ChannelHandler {
         let wsRequest = WSServerRequest(request: request)
+        let hasNoExtensions: Bool = hasNoExtensionsConfigured(request: request)
+        
         let service = registry[wsRequest.urlURL.path]
 
-        let connection = WebSocketConnection(request: wsRequest, service: service)
+        let connection = WebSocketConnection(request: wsRequest, service: service, hasNoExtensions: hasNoExtensions )
```

Comment:
`hasNoExtensions` seems to be fully derivable from `request`  - we could avoid passing it as an extra argument and move the check fully inside `WebSocketConnection`